README.md

```diff
@@ -1,22 +1,23 @@
 # CSK_Module_LicenseManager
-## INFO: Draft version. Not further developed / tested so far! Check for available GitHub forks of this repository to see latest updates.
 
 Module to provide SICK LicenseManager functionality.
 
-*If available, please also add a screenshot/gif of the UI of the module here placed within /docu/media/ (see code)*
 ![](./docu/media/UI_Screenshot.png)
 
 ## How to Run
 
-[***...please fill with informations...***]  
 For further information check out the [documentation](https://raw.githack.com/SICKAppSpaceCodingStarterKit/CSK_Module_LicenseManager/main/docu/CSK_Module_LicenseManager.html) in the folder "docu".
 
 ## Information
 
+**INFO for developers**  
+UI sources are not included in this repository.  
+
 Tested on:
 |Device|Firmware|Module version
```

Comment:
All modules:
This line is missing a "|" at the end. Nevertheless, the table has a closed frame maybe due to the other lines ending with "|".

Reply:
Done